autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/SchedulePreview.css

```diff
@@ -26,6 +26,7 @@
 .schedule-header {
     display: flex;
     align-items: flex-end;
+    margin-bottom: 1vh;
```

Comment:
Honestly I don't think this needs a margin, but I could be convinced otherwise. If we did do it, I don't think this needs a relative value - I think `8px` or so would do. To me it feels weird seeing the space between them grow when you resize the window.

Reply:
I think it looks more natural with some space there but then again I am used to the GPA being there. I'll change it to 8px for now.